recipes/greenlet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "greenlet" %}
+{% set version = "0.4.12" %}
+{% set sha256 = "e4c99c6010a5d153d481fdaf63b8a0782825c0721506d880403a3b9b82ae347e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```

Comment:
Please reset to `0`. All recipes at staged-recipes should start with build number `0`.

Note: When changing the version (to an unreleased one), one should reset to the build number to `0` as well.